Caddyfile.example

```diff
@@ -0,0 +1,5 @@
+localhost {
+      reverse_proxy host.docker.internal:3000
```

Comment:
Is this going to work on Linux? `host.docker.internal` used to be a MacOS thing. I know there were plans to add it to Linux versions as well, but I'm not sure about the status of it.

This would suggest that (as of end of last year) they implemented it for Linux, but it requires an additional flag when running (see the "accepted answer"):
https://stackoverflow.com/questions/48546124/what-is-the-linux-equivalent-of-host-docker-internal

Reply:
FYI - Not a reason to hold the PR IMO. But if it an actual problem it could become a new ticket.

Reply:
That is a good catch @ammazzaw. I'll test on my Linux laptop.

Reply:
Tested and works on my Linux laptop. Also, see Jawira's answer on this Stackoverflow question: https://stackoverflow.com/questions/48546124/what-is-the-linux-equivalent-of-host-docker-internal/67158212#67158212